kaldb/src/main/java/com/slack/kaldb/chunkManager/CachingChunkManager.java

```diff
@@ -86,6 +90,10 @@ protected void shutDown() throws Exception {
           }
         });
 
+    replicaMetadataStore.close();
```

Comment:
These should be closed in reverse order to initialization.

Reply:
Done.